gradle.properties

```diff
@@ -25,8 +25,8 @@ slf4jApiVersion=1.7.25
 sparkVersion=3.0.1
 
 # Version and base tags can be overridden at build time.
-connectorVersion=0.9.0-SNAPSHOT
-pravegaVersion=0.9.0-2725.77b1398-SNAPSHOT
+connectorVersion=0.10.0-SNAPSHOT
+pravegaVersion=0.10.0-1.a621c55-SNAPSHOT
```

Comment:
We can use this `0.10.0-2869.842cbda-SNAPSHOT` now, the commit number goes back to normal after my fix.

Reply:
Updated.